lib/auth/MyPadsAuthProvider.js

```diff
@@ -23,19 +42,15 @@ class MyPadsAuthProvider {
         this.token = login.token;
         this.userId = login.user._id;
         if (!this.token) return;
-        window.localStorage.setItem('write_access_token', this.token);
         await this.syncAllPads(this.token);
+        this.validateToken();
```

Comment:
Do you think this call is really necessary?
After all we -just- logged in in the lines above; I think it's safe to assume that the token is valid at this point and we can just return it.

Reply:
ok, deleted it.